<launch>
    <arg name="input_topic" default="/local_metric_map/intermediate_point_cloud" />
    <arg name="ptu_tilt_motor_frame" default="/ptu_tilt_motor" />
    <arg name="ptu_hinge_frame" default="/ptu_hinge" />
    <arg name="semantic_config_file" default="$(find semantic_segmentation)/data/semantic.conf" />
    <arg name="semantic_forest_model" default="$(find semantic_segmentation)/data/rdf.dat" />
    <node pkg="semantic_segmentation" type="semantic_segmentation" name="semantic_segmentation" output="screen" >
        <param name="input_topic" value="$(arg input_topic)" type="string" />
        <param name="ptu_tilt_motor_frame" value="$(arg ptu_tilt_motor_frame)" type="string" />
        <param name="ptu_hinge_frame" value="$(arg ptu_hinge_frame)" type="string" />
        <param name="semantic_config_file" value="$(arg semantic_config_file)" type="string" />
        <param name="semantic_forest_model" value="$(arg semantic_forest_model)" type="string" />
    </node>
</launch>
